app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -25,41 +25,9 @@ keywords:
     - Grants
     - Financial Support
     - Undergraduate
-    - Troops to Teachers
-    - Army
-    - Air Force
-    - RAF
-    - Navy
-    - Military
 ---
```

Comment:
Line 6: might be worth adding in `postgraduate` to the meta description now that you've removed undergraduate and veteran bursaries from this page?